src/server/package.json

```diff
@@ -17,6 +17,9 @@
   },
```

Comment:
And I guess we would be needing a common package.json for both React and Node. not just for node